web/ui/module/codemirror-promql/MAINTAINERS.md

```diff
@@ -0,0 +1,2 @@
+* Augustin Husson <husson.augustin@gmail.com> [@nexucis](https://github.com/Nexucis)
```

Comment:
Probably this file is not needed anymore indeed, even for the mirror.
But if i'm putting my name in the MAINTAINERS root file, it can be weird since I'm not part of Prometheus right ?

Reply:
done

Reply:
should I modify the file https://github.com/prometheus/prometheus/blob/main/.github/CODEOWNERS too ?